rpcs3/Emu/Memory/vm_locking.h

```diff
@@ -76,7 +77,7 @@ namespace vm
 		}
 
 		// Fallback to slow path
-		range_lock_internal(range_lock, begin, size);
+		range_lock_internal(range_lock, begin, size, touch_mem);
```

Comment:
Ideally, we don't need slow path at all for checking reservation.
If our range is locked, can assume reservation loss.
If our range is allocated/deallocated, can assume reservation loss as well.

Reply:
range_lock_internal is not called anymore fro it.